### PR TITLE
Vs feature faved context

### PR DESCRIPTION
**Pull request by verasunyy**
-add fave store item brand models
-add fave store item brand context

### CODE REVIEW

File: client/js/components/StoreCard/StoreCard.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                <Text >Remove From Faves</Text>
+              </TouchableOpacity>
+              <TouchableOpacity
+                onPress={() => createFaveStore(store.id)}
+                activeOpacity={0.5}
+              >
+
+                <Text>Add To Faves</Text>
+              </TouchableOpacity>
+              {/* <Text>{store.storeLogo}</Text> */}
```
Comment (siddharthparmar7) [inline client/js/components/StoreCard/StoreCard.js]: remove commented code.

File: client/js/screens/StoresBrands/StoresBrandsContainer.js [hunk 2]
```diff
@@ -1,13 +1,74 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import StoresBrands from './StoresBrands';
-import {withNavigation} from 'react-navigation';
+// import { withNavigation } from 'react-navigation';
```
Comment (siddharthparmar7) [inline client/js/screens/StoresBrands/StoresBrandsContainer.js:3]: remove commented code.

File: client/react-native.config.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
-module.exports = {
-  assets: ['./js/assets/fonts'],
-};
+// module.exports = {
+//   assets: ['./js/assets/fonts'],
+// };
```
Comment (siddharthparmar7) [inline client/react-native.config.js:3]: remove commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts. Thank you.

File: client/js/components/StoreCard/StoreCard.js [hunk 4]
```diff
@@ -1,29 +1,54 @@
...
+                <View style={styles.storeContentContainer}>
+                  {/* <Text>{store.title}</Text>
         <Text>{store.address}</Text>
         <Text>{store.phone}</Text> */}
-        <Text>Store Title</Text>
-        <Text>Store Address</Text>
-        <Text>Store Phone</Text>
-      </View>
-    </View>
+                  <Text>Store Title</Text>
```
Comment (siddharthparmar7) [inline client/js/components/StoreCard/StoreCard.js]: remove unnecessary comments.

File: client/react-native.config.js (hunk 3 above)
Comment (siddharthparmar7) [inline client/react-native.config.js:3]: are you removing fonts?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/components/StoreCard/StoreCard.js (hunk 4 above)
Comment (verasunyy) [reply client/js/components/StoreCard/StoreCard.js]: done, <Image/> source is for testing

File: client/react-native.config.js (hunk 3 above)
Comment (verasunyy) [reply client/react-native.config.js:3]: this is from the develop

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**